doc/OnlineDocs/modeling_extensions/gdp.rst

```diff
@@ -61,32 +60,36 @@ the example returns.
 Transformation
 --------------
 
-In order to use the solvers currently available, one must convert the
-disjunctive model to a standard MIP/MINLP model.  The easiest way to do
-that is using the (included) BigM or Convex Hull transformations.  From
-the Pyomo command line, include the option ``--transform
-pyomo.gdp.bigm`` or ``--transform pyomo.gdp.chull``
+To use standard commercial solvers, you must convert the disjunctive model to a standard MIP/MINLP model.
+The two classical strategies for doing so are the (included) Big-M and Hull reformulations.
+From the Pyomo command line,  include the option ``--transform pyomo.gdp.bigm`` or ``--transform pyomo.gdp.chull``.
+If you are using a Python script, ``TransformationFactory`` accomplishes the same functionality:
 
-Notes
------
+- ``TransformationFactory('gdp.bigm').apply_to(model)``
+- ``TransformationFactory('gdp.chull').apply_to(model)``
+
+.. note::
 
-Some notes:
+    - all variables that appear in disjuncts need upper and lower bounds
```

Comment:
This is only true for thc HUll reformulation.  BigM only needs bounds if you are going to let us automatically determine M

Reply:
Even for nonlinear constraints?

Reply:
Correct.  BigM will happily relax the constraints without regard to the validity of the relaxation (if you provide an M value, we trust it is correct).